### PR TITLE
Make window urgent when bell rings on inactive tab

### PR DESCRIPTION
**Pull request by dbeniamine**
Hi,

Thanks for your excelent plugin !
I have added the possiblity to make urxvt window urgent when the bell rings on an inactive tab. This functionality can be activated with the following boolean:

URxvt.tabbed.tab-UrgentOnBell: true

Bye,

David

### CODE REVIEW

File: tabbedex [hunk 1]
```diff
@@ -575,6 +576,7 @@ sub on_init {
    $self->{autohide} = $self->rs_bool ('autohide', 'false');
    $self->{no_default_keys} = $self->rs_bool ('no-tabbedex-keys', 0);
    $self->{reopen_on_close} = $self->rs_bool ('reopen-on-close', 0);
+   $self->{tab_UrgentOnBell} = $self->rs_bool ('tab-UrgentOnBell', 0);
```
Comment (mina86) [inline tabbedex]: $self->{tab_urgent_on_bell} = $self->rs_bool ('tab-urgent-on-bell', 0);

File: tabbedex [hunk 2]
```diff
@@ -181,6 +181,7 @@ sub tab_bell_active ($$$) {
         if ($tab == $self->{cur}) {
             return $now - $tab->{bell_rung} < $self->{bell_timeout};
         } else {
+            $self->{term}->set_urgency(1) if ($self->{tab_UrgentOnBell});
```
Comment (mina86) [inline tabbedex]: I’d rather this was a proper if (condition) { action; }.

**Comment by dbeniamine** [comment]
Done

**Comment by mina86** [comment]
So actually, why is it only for inactive tabs?

**Comment by dbeniamine** [comment]
If you set `Urxvt*UrgentOnBell`, the active tab will be made urgent by Urxvt on bell. But with tabbedex this wasn't impacting inactive tabs, thus this pull request.

**Comment by mina86** [comment]
Should tabbedex instead just read URxvt.UrgentOnBell then?

**Comment by dbeniamine** [comment]
I'm not sure, I guessed that If no one have done this pull request yet, it might means that some people are happy with the behavior where only the active tab can make the window urgent.

**Comment by mina86** [comment]
There’s relatively high bar for making such a pull request: you must have urgentOnBell enable, you must notice it doesn’t affect inactive tabs, you must care enough to look for a solution, you must find the solution and you must do the pull. I’ve pushed a commit getting rid of the ‘urgent-on-bell’ in favour of just respecting urxvt’s urgentOnBell instead. Let’s see how this works out.

**Comment by dbeniamine** [comment]
You're right, I didn't see it that way.

**Pull request closed without merging**